src/main/java/org/aksw/gerbil/database/ExperimentDAOImpl.java

```diff
@@ -88,52 +88,33 @@ public ExperimentDAOImpl(DataSource dataSource, long resultDurability) {
 
     @Override
     public List<ExperimentTaskResult> getResultsOfExperiment(String experimentId) {
+    	
         MapSqlParameterSource parameters = new MapSqlParameterSource();
         parameters.addValue("id", experimentId);
+        
         List<ExperimentTaskResult> result = this.template.query(GET_EXPERIMENT_RESULTS, parameters,
                 new ExperimentTaskResultRowMapper());
         // FIXME remove this ugly workaround regarding the version of an
         // experiment task
         for (ExperimentTaskResult e : result) {
-            addVersion(e);
+            getVersion(e);
```

Comment:
@CatherineRomiyo didn't we wanted to get rid of this additional method? It would be much better to extend `GET_EXPERIMENT_RESULTS` to select the version and set this information with adapting the ExperimentTaskResultRowMapper, right? :wink: